change/@fluentui-react-badge-16df5a7e-d608-42e5-803b-981231c94c7a.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Updated badge descriptions to be specific.",
```

We should add the correct label:

```suggestion
  "comment": "docs: Updated badge descriptions to be specific.",
```